x-pack/plugins/enterprise_search/public/applications/applications/components/engine/engine_connect/search_application_documentation.tsx

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiImage,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import welcomeGraphicLight from '../../../../../assets/images/welcome_light.svg';
+
+export const SearchApplicationDocumentation = () => {
+  return (
+    <EuiPanel color="transparent">
+      <EuiFlexGroup justifyContent="spaceBetween" direction="row">
+        <EuiFlexGroup direction="column">
+          <EuiImage size="l" float="left" src={welcomeGraphicLight} alt="" />
+
+          <EuiText>
+            <h2>
+              {i18n.translate(
+                'xpack.enterpriseSearch.content.searchApplication.documentation.text.title',
+                {
+                  defaultMessage: 'Learn more about Search Applications',
+                }
+              )}
+            </h2>
+          </EuiText>
+
+          <EuiText>
+            <p>
+              {i18n.translate(
+                'xpack.enterpriseSearch.content.searchApplication.documentation.text.paragraph',
+                {
+                  defaultMessage:
+                    'Search Applications help make your Elasticsearch data searchable for end users, with out-of-the-box relevance, analytics and personalization tools.',
```

Comment:
```suggestion
                    'Search Applications help make your Elasticsearch data easily searchable for end users.',
```

We had to update this line elsewhere because analytics are not hooked up to Search Applications in 8.8.0. @julianrosado